client/src/test/java/com/paypal/selion/mobile/sample/TestTablePage.java

```diff
@@ -0,0 +1,77 @@
+/*-------------------------------------------------------------------------------------------------------------------*\
+|  Copyright (C) 2016 PayPal                                                                                          |
+|                                                                                                                     |
+|  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance     |
+|  with the License.                                                                                                  |
+|                                                                                                                     |
+|  You may obtain a copy of the License at                                                                            |
+|                                                                                                                     |
+|       http://www.apache.org/licenses/LICENSE-2.0                                                                    |
+|                                                                                                                     |
+|  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed   |
+|  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for  |
+|  the specific language governing permissions and limitations under the License.                                     |
+\*-------------------------------------------------------------------------------------------------------------------*/
+
+package com.paypal.selion.mobile.sample;
+
+import com.paypal.selion.annotations.MobileTest;
+import com.paypal.selion.internal.platform.grid.WebDriverPlatform;
+import com.paypal.selion.mobile.sample.pages.*;
+import com.paypal.selion.platform.mobile.UIOperationFailedException;
+import com.paypal.selion.platform.mobile.android.UiList;
+
+import org.openqa.selenium.By;
+import org.testng.annotations.Test;
+
+public class TestTablePage {
+
+    @MobileTest
+    @Test
+    public void testTableClickAndroid() throws Exception {
```

Comment:
@mach6 This test will work on both platforms. The method has a wrong name. i will fix it in next commit.